src/App.tsx

```diff
@@ -1,14 +1,35 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './services/getTodos';
+import { FilteringType } from './types/FilteringType';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosLoading, setTodosLoading] = useState(false);
+  const [filter, setFilter] = useState<FilteringType>(FilteringType.all);
+  const [title, setTitle] = useState('');
+  const [todoCard, setTodoCard] = useState<Todo>();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [iconId, setIconId] = useState(0);
+
+  useEffect(() => {
+    setTodosLoading(true);
+    setTimeout(() => {
+      getTodos().then((todo) => {
```

Comment:
You already have ready-made functions for requests in the API folder
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/80700247/1b477c50-2de6-468f-9f3c-6e2cc8c66b6a)